src/parser.js

```diff
@@ -386,24 +437,106 @@ export default class Parser extends Stream {
             },
             'skip'() {
               this.manifest.skip = entry.attributes;
+
+              if (!entry.attributes.hasOwnProperty('SKIPPED-SEGMENTS')) {
+                this.trigger('warn', {
```

Comment:
Should we trigger an error for things like this??

Reply:
As long as triggering an error won't cause us to stop parsing or something, we should, otherwise, I think we can continue with a warning.

Reply:
I think we should continue with a warning unless we find out that this stops us from playing a source back.

Reply:
Sounds good.